popsycle/synthetic.py

```diff
@@ -560,56 +619,71 @@ def calc_current_initial_ratio(iso_dir, out_file='current_initial_stellar_mass_r
+                 'ukirt,K', 'ukirt,J']
+
     # Run all the clusters for the different ages in logage_vec
     for i in range(len(logage_vec)):
-        my_iso = synthetic.IsochronePhot(logage_vec[i], 0, 10, 
-                                         evo_model = evolution.MISTv1(),
-                                         filters = filt_list,
-                                         iso_dir = iso_dir) # make isochrone
-
-        trunc_kroupa = imf.IMF_broken_powerlaw(mass_limits, powers) # define IMF
-        cluster = synthetic.ResolvedCluster(my_iso, trunc_kroupa, cluster_mass) # make cluster
+        my_iso = synthetic.IsochronePhot(logage_vec[i], 0, 10,
+                                         evo_model=evolution.MISTv1(),
+                                         filters=filt_list,
+                                         iso_dir=iso_dir)  # make isochrone
+
+        trunc_kroupa = imf.IMF_broken_powerlaw(mass_limits,
+                                               powers)  # define IMF
+        cluster = synthetic.ResolvedCluster(my_iso, trunc_kroupa, cluster_mass,
+                                            set_random_seed=debug)  # make cluster
         output = cluster.star_systems
-        
+
         # Find the stars in MIST not in Galaxia (i.e. WDs) and figure out how much mass they contribute
         bad_idx = np.where(output['phase'] == 101)[0]
-        bad_mass = np.sum(output['mass'][bad_idx]) # in m_sol
+        bad_mass = np.sum(output['mass'][bad_idx])  # in m_sol
 
         # Figure out the current mass (stars + post-AGB stars)
-        current_mass = np.sum(output['mass']) # in m_sol
-        
+        current_mass = np.sum(output['mass'])  # in m_sol
+
         # Figure out the current STELLAR mass (no post-AGB stars) This is what we want for the ratio
         # since Galaxia does NOT have white dwarfs.
         current_stellar_mass = current_mass - bad_mass
-    
+
         # Calculate the ratio of initial to final stellar mass.
-        current_initial_mass_ratio = current_stellar_mass/cluster_mass
+        current_initial_mass_ratio = current_stellar_mass / cluster_mass
         current_initial_ratio_array[i] = current_initial_mass_ratio
         print(current_initial_mass_ratio)
-        
+
     # Reshape vectors so they can be concatenated into two columns
-    logage_vec = logage_vec.reshape((len(logage_vec),1))
-    current_initial_ratio_array = current_initial_ratio_array.reshape((len(logage_vec),1))
-    tab = np.concatenate((logage_vec, current_initial_ratio_array), axis = 1)
+    logage_vec = logage_vec.reshape((len(logage_vec), 1))
+    current_initial_ratio_array = current_initial_ratio_array.reshape(
+        (len(logage_vec), 1))
+    tab = np.concatenate((logage_vec, current_initial_ratio_array), axis=1)
     np.savetxt(out_file, tab)
 
-    return 
+    return
+
 
-def current_initial_ratio(logage, ratio_file, iso_dir):
+def current_initial_ratio(logage, ratio_file, iso_dir, debug=False):
```

Comment:
I thought we were changing this to random_seed?

Reply:
Related: lines 303 - 309? Is that debug or random seed?

Reply:
See comment below